### PR TITLE
examples - stoploss & takeprofit - transpilable

### PR DESCRIPTION
**Pull request by ttodua**
I have removed the 'python-specific' example, and replaced it with a transpilable .ts example for 'stop-loss/take-profit' orders.
Sorry to mention this, but in my opinion, the existing python example was too complex for examplary purposes. I understand that it's good that we teach users about other generic ccxt functions, but imo, this is slightly out-of-scope for such examples, as people have to read around 100 lines to find out how to just execute 'stop-loss' order. In addition to our large readmes, we should try to avoid forcing people to read much code to find out things.
I've tried to minimize and make it compact, with some shorter (but clear) explanations, and anyone which has a basic knowledge of scripting/ccxt, would be able to understand it with less reading

### CODE REVIEW

File: examples/ts/create-order-position-with-takeprofit-stoploss.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// @ts-nocheck
```
Comment (carlosmiei) [inline examples/ts/create-order-position-with-takeprofit-stoploss.ts:1]: @ttodua the examples are not 100% equal, can we keep both for now?

Comment (ttodua) [reply examples/ts/create-order-position-with-takeprofit-stoploss.ts:1]: I think more files for same example would be unnecessary and would not benefitable for users, but oky as you wish, reverted & pushed.

Comment (carlosmiei) [reply examples/ts/create-order-position-with-takeprofit-stoploss.ts:1]: @ttodua it's okay as long as they're both correct

Comment (ttodua) [reply examples/ts/create-order-position-with-takeprofit-stoploss.ts:1]: i think now it's ready

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**